ste/sourceInfoProvider-File.go

```diff
@@ -157,5 +220,7 @@ func (p *fileSourceInfoProvider) GetFileLastModifiedTime() (time.Time, error) {
 		return time.Time{}, err
 	}
 
+	// We ignore smblastwrite because otherwise the tx will fail s2s
```

Comment:
During a previous point in the PR, I'd considered the SMB last write time as a part of the last modified time. Turns out, this can vary from the REST last write time, and cause an item that's not modified during transfer to think it is.